pypika/queries.py

```diff
@@ -366,6 +366,8 @@ def rollup(self, *terms, **kwargs):
             self._mysql_rollup = True
             self._groupbys += terms
 
+        elif 0 < len(self._groupbys) and isinstance(self._groupbys[-1], Rollup):
```

Comment:
I think a comment here would be handy to know the difference between this branch and the one below it.

Reply:
fixed

Reply:
Thanks